samples/xlsx-import+svelte/src/App.svelte

```diff
@@ -0,0 +1,88 @@
+<script lang="ts">
+    import { Workbook } from 'exceljs';
+    import { Importer } from 'xlsx-import/lib/Importer';
+    import XlsxImportLogo from './components/XlsxImportLogo/Logo.svelte';
+    import { Invoice, getInvoiceConfig } from './components/Invoice';
+    import 'tachyons/css/tachyons.min.css';
+
+    let invoice = null;
+
+    function onImportInvoice(importer: Importer) {
+        const {misc:miscCfg, seller:sellerCfg, buyer:buyerCfg, items:itemsCfg} = getInvoiceConfig();
+
+        const { date, dueDate } = importer.getAllItems(miscCfg)[0];
```

Comment:
Well I found how to fix it. Sort of

Reply:
Right, I checked the validate command, and it seems to work:

![image](https://user-images.githubusercontent.com/2893099/111549182-0dabf600-877c-11eb-8ab6-067fbff4de70.png)

Also, I confirm that WebStorm lost types - probably lack good integration.

Reply:
I consider about one of two available ways. Which one would you like to prefer? (and why?)

1. Keep it as is. TS types work for `npm run validate`. However, IDE doesn't handle type at all.

2. Reject a Typescript and use only plain Javascript. Then types will not be a problem (because they don't exist ;) ). I favor this one because I believe the experience from working code properly and IDE will be much better than the fact that we use TS (for this example, ofc.).

For me, doesn't working types in IDE is pointless and probably will make a bad filling for someone interested. I'd rather leave TypeScript in favor of JavaScript. Before I make a decision, I really want to know what you are thinking about it.

Reply:
Personally I prefer guarantees over good UI\UX
Since I'm not an expert in Svelte I'll try to ask in community how to solve this problem more elegantly. Maybe there's some solution
If there's no such solution I'll remove TS

Reply:
ok, thank you

Reply:
We found this solution with explicit type import in App.svelte
What do you think?

Reply:
Ok, I tested it, it works great. Good job